tests/sharedtria/mg_dof_01.cc

```diff
@@ -131,7 +131,7 @@ test()
         if (cell->center()[0] < 0.5 && cell->center()[1] < 0.5)
           cell->set_refine_flag();
       if (dim == 3)
-        if (cell->center()[0] < 0.5 && cell->center()[1] &&
```

Comment:
The original version is probably wrong, but this is a behavior change. Does the test still pass?

Reply:
Yes, the test suite passes with my configuration.